Doc/library/logging.rst

```diff
@@ -562,7 +562,7 @@ subclasses. However, the :meth:`!__init__` method in subclasses needs to call
 
    .. method:: Handler.setFormatter(fmt)
 
-      Sets the :class:`Formatter` for this handler to *fmt*.
+      Sets the :class:`Formatter` for this handler to *fmt*. The *fmt* argument must be an initialized :class:`Formatter` instance or ``None``.
```

Comment:
```suggestion
      Sets the :class:`Formatter` for this handler to *fmt*.
      The *fmt* argument must be a :class:`Formatter` instance or ``None``.
```

Just to avoid a too long line (80-chars max). Btw, what do you mean by "initialized" instance? Once it's created, the instance is ready for use right?